app/services/manual/publish_service.rb

```diff
@@ -10,7 +10,7 @@ def initialize(user:, manual_id:, version_number:)
   def call
     if version_number == manual.version_number
       manual.publish
-      log_publication
+      PublicationLogger.new.call(manual)
```

Comment:
Minor: I might be tempted to take this a step further and build the `PublicationLogger` in the service constructor.